shells/dev-shell/output-pane.js

```diff
@@ -186,23 +201,13 @@ class ArcPanel extends HTMLElement {
     this.linkedArc = arc;
   }
 
-  display(description) {
-    this.serialControl.style.display = 'inline-block';
-    if (this.linkedArc._stores.length > 0) {
-      this.storesControl.style.display = 'inline-block';
-    }
+  async display(description) {
```

Comment:
why async?

Reply:
Because of the `await storePanel.attach` (which is async because it ends up reading from the store to populate the data).

... but the real question is "why is this called 'display' which would suggest it just displays a description, when in reality it does a bunch of other post-instantiation things?" And that's a good question! I'll rename the method.